include/arch/armv7m/stdint.h

```diff
@@ -16,19 +16,21 @@
 #ifndef _LIBPHOENIX_ARCH_ARMV7M_STDINT_H_
 #define _LIBPHOENIX_ARCH_ARMV7M_STDINT_H_
 
+#include <phoenix/arch/armv7m/stdtypes.h>
+
 /* The following sections refer to ISO/IEC 9899:1999 */
 
 /* 7.18.1.1 Exact-width integer types */
 
-typedef signed char int8_t;
-typedef short int16_t;
-typedef int int32_t;
-typedef long long int64_t;
+typedef __s8 int8_t;
+typedef __s16 int16_t;
+typedef __s32 int32_t;
+typedef __s64 int64_t;
 
-typedef unsigned char uint8_t;
-typedef unsigned short uint16_t;
-typedef unsigned int uint32_t;
-typedef unsigned long long uint64_t;
+typedef __u8 uint8_t;
+typedef __u16 uint16_t;
+typedef __u32 uint32_t;
+typedef __u64 uint64_t;
```

Comment:
Potentially it  could be moved to generic include/stdint.h 

Then we may want to not used it in least/fast/ptr/max types  defined below or firstly typedef and then include.

side note: least type could also be moved to generic include/stdint.h in section 
 USE_STANDARD_TYPES_STDINT